src/experimentation-opportunities/experimentation-opportunities.js

```diff
@@ -11,16 +11,252 @@
+      // eslint-disable-next-line no-param-reassign
+      opportunites[index] = oppty;
+    }
+  }
+}
+
+/* c8 ignore stop */
+
+async function createOrUpdateOpportunityEntity(opportunity, context, existingOpportunities) {
+  const { log, dataAccess } = context;
+  const { Opportunity } = dataAccess;
+  const existingOpportunity = existingOpportunities.find(
+    (oppty) => (oppty.getType() === opportunity.type) && oppty.getData()
+    && (oppty.getData().page === opportunity.data.page),
+  );
+  if (existingOpportunity) {
+    if (existingOpportunity.getStatus() === 'NEW') {
+      // remove and create a new opportunity entity with new data
+      log.info(`[${opportunity.type}] Opportunity entity with status: NEW for ${opportunity.data.page} exists, so removing it and creating a new opportunity entity`);
+      await existingOpportunity.remove();
+    } else {
+      log.info(`[${opportunity.type}] Opportunity entity already exists with status: ${existingOpportunity.getStatus()} for ${opportunity.data.page}, so skipping`);
+      return false;
+    }
+  }
+  await Opportunity.create(opportunity);
+  return true;
+}
+
+function convertToOpportunityEntity(oppty, auditData) {
+  return {
+    siteId: auditData.siteId,
+    auditId: auditData.id,
+    runbook: 'https://adobe.sharepoint.com/:w:/r/sites/aemsites-engineering/_layouts/15/Doc.aspx?sourcedoc=%7B19613D9B-93D4-4112-B7C8-DBE0D9DCC55B%7D&file=Experience_Success_Studio_High_Organic_Traffic_Low_CTR_Runbook.docx&action=default&mobileredirect=true',
+    type: 'high-organic-low-ctr',
+    origin: 'AUTOMATION',
+    title: 'page with high organic traffic but low click through rate detected',
+    description: 'Adjusting the wording, images and/or layout on the page to resonate more with a specific audience should increase the overall engagement on the page and ultimately bump conversion.',
+    status: 'NEW',
+    guidance: {
+      recommendations: oppty.recommendations,
+    },
+    tags: ['Engagement'],
+    data: {
+      page: oppty.page,
+      pageViews: oppty.pageViews,
+      samples: oppty.samples,
+      screenshot: oppty.screenshot,
+      trackedKPISiteAverage: oppty.trackedKPISiteAverage,
+      trackedPageKPIName: oppty.trackedPageKPIName,
+      trackedPageKPIValue: oppty.trackedPageKPIValue,
+      opportunityImpact: oppty.opportunityImpact,
+      metrics: oppty.metrics,
+    },
+  };
+}
+
+export async function postProcessor(auditUrl, auditData, context) {
+  const { log } = context;
+  const { dataAccess } = context;
+  let updatedEntities = 0;
+  log.info(`Experimentation Opportunities post processing started for ${auditUrl} from audit ${auditData.id}`);
+  const existingOpportunities = await dataAccess.Opportunity.allBySiteId(auditData.siteId);
+
+  // Get opportunities with recommendations
+  const opportunities = auditData.auditResult.experimentationOpportunities
+    .filter((oppty) => oppty.type === 'high-organic-low-ctr' && oppty.recommendations);
```

Comment:
I'd say it is best to namespace the oppty type such as `experimentation-high-organic-low-ctr` because other oppties are going to use the same type field such as `cwv`...

Reply:
@ekremney didn't get it. Why would `cwv` opportunities will be using type as `high-organic-low-ctr` ? Shouldn't those be `cwv` opportunities?